### DISCUSSION TITLE
Cannot find color resources

### DISCUSSION THREAD

**Discussion by hxhgxy** [MahApps/MahApps.Metro#4415] [Q&A]
Cannot find color resources - GeneratorParameters.json didn't work.

What shall I do? Thanks.

**Comment by timunie** [comment]
Try to clean generated files. Check VS docs for how to do it.

**Comment by hxhgxy** [comment]
I download the source codes from github a moment ago and then run it...

**Reply by timunie** [reply]
Check if your VS version is up to date: https://github.com/MahApps/MahApps.Metro/wiki/Building-the-MahApps.Metro-solution

**Comment by hxhgxy** [comment]
VS professional 2022 v17.6.5
